### PR TITLE
Integrate Portfolio page with Netlify CMS data 

### LINKED ISSUE #67: Integrate Portfolio page with Netlify CMS data

### PR DESCRIPTION
**Pull request by mecies**
## Description

Setup for reading data from markdown in page components and PortfolioPage with data read from markdown.

Display from markdown files:
- [x] Portfolio page
- [x] Developer profile

This PR covers:
- Removing stories for Portfolio page`s (they were causing annoying errors after GraphQL setup) and they are pretty redundant after the page is done
- Displaying data on the `portfolio-page` from markdown files
- Fixing `PortfolioProjectDialog` component to accept missing props
- Adding `dayjs` library to format dates
- Fixing height in CardDesktop component 

Closes #67 

### Desktop
Screenshot 2020-09-04 at 21 31 35
Screenshot 2020-09-04 at 21 31 40
Screenshot 2020-09-04 at 21 31 49
Screenshot 2020-09-04 at 21 31 54

### Tablet
Screenshot 2020-09-04 at 21 33 20
Screenshot 2020-09-04 at 21 33 32
Screenshot 2020-09-04 at 21 33 37

### Mobile
Screenshot 2020-09-04 at 21 33 53
Screenshot 2020-09-04 at 21 34 02
Screenshot 2020-09-04 at 21 34 07

### CODE REVIEW

File: src/pages/portfolio-page/index.tsx [hunk 1]
```diff
@@ -153,3 +203,53 @@ const PortfolioPage = () => {
...
+            publicURL
+          }
+          project_technologies {
+            technology_name
+          }
+          project_name
+        }
+      }
+    }
+    developerProfile: markdownRemark(fileAbsolutePath: { regex: "/developer-profile/index-1.md/" }) {
```
Comment (KlaraGajaszek) [inline src/pages/portfolio-page/index.tsx]: DetailsCard perform in more than one page component, so maybe you can move this 
query to other query or use useStaticQuery directly in the DetailsCard component and then everybody will use the same query and we won't duplicate our code ?

Comment (mecies) [reply src/pages/portfolio-page/index.tsx]: Yeah, I think that it's a great idea, I will try to use it directly in component 👍

**Review by mecies — COMMENTED** [review]
I have an error related to storybook and usage of static query and not sure how to fix it
https://www.gatsbyjs.com/docs/visual-testing-with-storybook/

File: src/components/PortfolioProject/PortfolioProjectDesktop.tsx [hunk 2]
```diff
@@ -59,24 +59,24 @@ const useStyles = makeStyles((theme) => ({
   contentMain: { padding: theme.spacing(3, 3, 1.5, 3) },
 }));
 
-const BlogPostDialogDesktop: FC<PortfolioProjectDialogProps> = ({ isOpen, ...props }) => {
+const BlogPostDialogDesktop: FC<PortfolioProjectDialogProps> = ({ ...props }) => {
```
Comment (KonradSzwarc) [inline src/components/PortfolioProject/PortfolioProjectDesktop.tsx]: You don't need to destructure props here

File: src/components/PortfolioProject/PortfolioProject.stories.tsx [hunk 3]
```diff
@@ -20,6 +23,10 @@ const basePropsData = {
     'Nullam vel eros non arcu feugiat lobortis. Quisque a ultrices neque. Maecenas a ultrices sapien, sit amet aliquam est. Nunc porttitor lorem purus, non venenatis magna suscipit a. Nullam pellentesque, lacus a accumsan pretium, justo massa commodo nulla, ut vulputate elit justo eget neque. Donec aliquet dui quis nisi mattis, nec ultricies nulla elementum. Aenean in nisl eget metus elementum aliquam.',
   contentMainRole:
     'Sed eu enim auctor, auctor ipsum id, finibus metus. Donec vel efficitur mi. Donec at justo nunc. Nullam vitae urna ut leo vehicula sodales. Vestibulum ut dui laoreet, consequat mi a, aliquam quam. Vivamus ac felis non lectus dapibus pulvinar. Nulla imperdiet maximus mi vel lacinia. Sed sagittis velit sed arcu hendrerit sollicitudin. Quisque faucibus bibendum egestas.',
+  tags: [{ name: 'Redux' }, { name: 'React' }, { name: 'TypeScript' }],
+  handleClose: () => window.alert('closed'),
```
Comment (KonradSzwarc) [inline src/components/PortfolioProject/PortfolioProject.stories.tsx]: As it's a Storybook maybe using an action would be a better choice for logging events?

File: src/components/PortfolioProject/PortfolioProjectTablet.tsx [hunk 4]
```diff
@@ -60,24 +60,24 @@ const useStyles = makeStyles((theme) => ({
   contentMain: { padding: theme.spacing(3, 3, 1.5, 3) },
 }));
 
-const BlogPostDialogTablet: FC<PortfolioProjectDialogProps> = ({ isOpen, ...props }) => {
+const BlogPostDialogTablet: FC<PortfolioProjectDialogProps> = ({ ...props }) => {
```
Comment (KonradSzwarc) [inline src/components/PortfolioProject/PortfolioProjectTablet.tsx]: No need for destructuring

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

File: src/components/PortfolioProject/PortfolioProject.stories.tsx [hunk 5]
```diff
@@ -20,6 +24,10 @@ const basePropsData = {
     'Nullam vel eros non arcu feugiat lobortis. Quisque a ultrices neque. Maecenas a ultrices sapien, sit amet aliquam est. Nunc porttitor lorem purus, non venenatis magna suscipit a. Nullam pellentesque, lacus a accumsan pretium, justo massa commodo nulla, ut vulputate elit justo eget neque. Donec aliquet dui quis nisi mattis, nec ultricies nulla elementum. Aenean in nisl eget metus elementum aliquam.',
   contentMainRole:
     'Sed eu enim auctor, auctor ipsum id, finibus metus. Donec vel efficitur mi. Donec at justo nunc. Nullam vitae urna ut leo vehicula sodales. Vestibulum ut dui laoreet, consequat mi a, aliquam quam. Vivamus ac felis non lectus dapibus pulvinar. Nulla imperdiet maximus mi vel lacinia. Sed sagittis velit sed arcu hendrerit sollicitudin. Quisque faucibus bibendum egestas.',
+  tags: [{ name: 'Redux' }, { name: 'React' }, { name: 'TypeScript' }],
+  handleClose: action('onClick'),
```
Comment (KonradSzwarc) [inline src/components/PortfolioProject/PortfolioProject.stories.tsx]: 3 times onClick? 😉

**Review by KonradSzwarc — APPROVED** [review]

File: src/components/PortfolioProject/PortfolioProject.stories.tsx (hunk 5 above)
Comment (mecies) [reply src/components/PortfolioProject/PortfolioProject.stories.tsx]: Sorrry for being lazy 🤣

**Review by maciekkwiet — APPROVED** [review]

**Review by KlaraGajaszek — APPROVED** [review]

**Pull request merged**